db/kv_dense_channel_storage.go

```diff
@@ -175,6 +184,7 @@ func (l *DenseBlockList) AddBlock() (*DenseBlock, error) {
 	}
 	casOut, err := l.indexBucket.WriteCas(l.activeKey, 0, 0, l.activeCas, storageValue, 0)
 	if err != nil {
+		base.LogTo("ChannelStorage+", "DenseBlockList %s got CAS error trying to persist to bucket.  Reloading and retrying", l)
```

Comment:
As above for info logging/AsZero.  If you think logging the other information on the DenseBlockList is valuable, you could remove the isZero from String()

Reply:
Removed `isZero()` from `String()`.